app/views/articles/show.html.haml

```diff
@@ -1,27 +1,31 @@
-.container.flex--2__2-1
-  %article{ tabindex: '0', id: 'content' }
-    %header
-      %h1= sanitize @article.title
-      = MarkdownHelper.markdown(sanitize @article.article_summary).html_safe
+.container
+  .article--wrapper
+    %article{ id: 'content', tabindex: '0' }
+      %header
+        - if @collections.any?
+          = render partial: 'collections/delimited_links', locals: { collections: @collections }
 
-    %div{ role: 'main' }
-      = MarkdownHelper.markdown(sanitize @article.article_body).html_safe
+        %h1= title(sanitize(@article.title))
+        = MarkdownHelper.markdown(sanitize(@article.article_summary)).html_safe
 
-  %aside
-    - if @related_articles.any?
-      .block
-        %h2= t('.related_articles')
-        %ul.list--pipe
-          - @related_articles.each do |article|
-            %li
-              .list--details
-                = link_to(sanitize(article.article_title), article_path(article.graph_id))
+      %div{ role: 'main' }
+        = MarkdownHelper.markdown(sanitize(@article.article_body)).html_safe
```

Comment:
How sure are we that sanitise is doing what we want here - `.html_safe` is VERY risky in this context